sql-cli/examples/.airflow/dags/simple.py

```diff
@@ -6,15 +6,15 @@
 from astro import sql as aql
 from astro.table import Table
 
-DAGS_DIRECTORY = Path(__file__).parent.as_posix()
+PROJECT_DIRECTORY = Path(__file__).parent.parent.parent.as_posix()
```

Comment:
I'm fine with this for an 0.1, but it feels weird to have this level of directory structure dependence. Like this would ONLY work if we push the entire project as is to the airflow deployment.

Reply:
No, from the generator perspective it should be sufficient to deploy `.airflow` directory which contains `dags` and `.sql`.

Reply:
This path resolution would still work like this, because the first parent of the dag file is `dags`, the 2nd is `.airflow` and the third is whatever you copy the directory to.